packaging/generic-deb/debian/amazon-ecs-init.install

```diff
@@ -0,0 +1,2 @@
+amazon-ecs-init usr/libexec/
```

Comment:
It doesn't look like you've listed /usr/libexec in the `debian/dirs` file.

Reply:
done